test/ParseableInterface/ObjC.swiftinterface

```diff
@@ -29,10 +29,9 @@ public class SomeClassInlinable {
     @inlinable get { return 0 }
     @inlinable set {}
   }
-  @objc @inlinable deinit {
-    print("bye")
-  }
+  @objc deinit
```

Comment:
@jrose-apple `@objc` is a no-op on a deinit isn't it? Should we warn and remove the attribute?

Reply:
This is how it gets printed right now, so we have to handle reading it back in.

Reply:
Sure, so it would be a warning at best. I just think it might confuse people ("do I need to @objc my deinit if my class is @objc, or otherwise some weird edge case won't work?" "No, it actually does nothing")

Reply:
\*shrug\* A warning sounds fine. I'd also be happy if we didn't print it here.